backend/src/Types/Database.hs

```diff
@@ -49,85 +49,98 @@ share
     ratingAfter Rating
     deriving Show
 
-   PlayerStats
```

Comment:
I broke `PlayerStats` into two tables, `PlayerStatsYear` for yearly stats, keyed on `(PlayerId, Year)`
and `PlayerStatsTotal` for the all-time stuff, keyed only on `PlayerId`

Reply:
Certainly possible, but I think my concern is around query efficiency. The leaderboard needs to compute totals for every player every time. That said, it still might work. We should discuss 💭